installer/cmd/root.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"os"
+	"os/user"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "gitpod-installer",
+	Short: "Installs Gitpod",
```

Comment:
Can you change this (and the line above) to something that is different from Gitpod's installer?

Reply:
Can we do that in a separate PR? I've already have #197 chained to this commit history... it will take an effort to get it right again 😅